app/1.0/docs/devguide/settings.md

```diff
@@ -35,5 +35,9 @@ Available settings:
     * `shady`. All local DOM is rendered using shady DOM, even where shadow DOM is supported (current default).
     * `shadow`. Local DOM is rendered using shadow DOM where supported (this will be the default in the future).
 
-*   `lazyRegister`—if true, many registration-time activities are deferred until the first instance of an element
-	type is created. Defaults to false. (This default may change in the future.)
+*   `lazyRegister`—options:
+    * `true`, many registration-time activities are deferred until the first instance of an element
+	type is created. Defaults to `false`. (This default may change in the future.)
+    * `"max"`, Defers all behavior work until first element creation. When setting `lazyRegister` to `"max"`, a behavior cannot setup an element's `is` or custom constructor via defining `factoryImpl`. Polymer calls `beforeRegister` on the prototype to preserve the ability to use it in ES6. 
```

Comment:
2nd sentence is a little confusing. Maybe: "cannot set an element's `is` property or create a custom constructor by defining the `factoryImpl` method. 

3rd sentence: calls `beforeRegister` when? Immediately?